README.md

```diff
@@ -1,2 +1,168 @@
+    <li>Redux Toolkit</li>
+    <li>React Router Dom v6</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Tools</summary>
+  <ul>
+    <li>VS Code</li>
+    <li>GIT</li>
+    <li>GITHUB</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+This project is a simple weather metrics app. The main features are:
+
+The app consists of three pages:
+
+- **The Home page displays a list of cities and the weather there.**
+- **The Details page displays the weather details for the selected city.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[N/A]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- In order to run this project you need:
+
+- [git](https://git-scm.com/downloads): A tool for managing source code
+- [Visual Studio Code](https://code.visualstudio.com/): A source code editor
+- [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer): A simple web server utility for Visual Studio Code
+
+### Setup
+
+Clone this repository to your desired folder:
+https://github.com/rachaouldbabaali/react-capstone
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/rachaouldbabaali/react-capstone
+```
+
+### Install
+
+Install this project with:
+
+` cd React-Redux-GP
+
+- Run npm install after cloning the repo
+
+### Usage
+
+- Run npm start for development.
+- Run npm run build for production
+
+### Run tests
+
+To run tests, run the following command:
+` npm run test`
+
+### Deployment
+
+You can deploy this project using:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [rachaouldbabaali](https://github.com/rachaouldbabaali)
+- Twitter: [@Racha_oba](https://twitter.com/Racha_oba)
+- LinkedIn: [Racha Ould Babaali](https://www.linkedin.com/in/rachaouldbabaali/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Improve the UI/UX design ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/rachaouldbabaali/react-capstone/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a start and share it with your friends.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to appreciate[Microverse](https://www.microverse.org/) for providing the resources and the platform for us to be a Professional full-stack developer.
+
```

Comment:
- [ ] Kindly consider giving appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).